context-ruby/spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative 'trunk_spec_helper'
```

Comment:
This is what's required to get uploads flowing from rspec directly. Behind the scenes it will automatically parse the results and publish them.